ansible/README.md

```diff
@@ -1,60 +1,13 @@
-OMERO VM Deployment
-===================
+OME Ansible 
+===========
 
-This contains example files for provisioning an OpenStack VM from scratch with OMERO using Ansible.
+This contains a variety of Ansible playbooks and roles, including example files for provisioning an OpenStack VM from scratch with OMERO using Ansible.
 Most of these scripts should also work on other platforms, providing the VM is brought up by some other method.
-The guest must be running CentOS 7.
 
+- Playbooks which start with `os` are OpenStack specific - the `os` stands for `openstack`.
+- Playbooks which start with `idr` are for the Image Data Repository.
+- Playbooks which start with `ci` are for OME continuous integration and build.
 
-Guide for the Impatient
------------------------
+HERE LINK TO: [[README-os.md]]
```

Comment:
This isn't resolving under https://github.com/kennethgillen/infrastructure/tree/bce516f7d509371e87cdedfcced7ce01e25e88bc/ansible